content/chat/connect.textile

```diff
@@ -0,0 +1,77 @@
+| closing | An explicit request by the developer to close the connection has been sent to the Ably service. If a reply is not received from Ably shortly, the connection will be forcibly terminated and the connection status will become closed. |
+| closed | The connection has been explicitly closed by the client. In the closed status, no reconnection attempts are made automatically by the SDK, and clients may not publish messages. |
+| failed | This status is entered if the SDK encounters a failure condition that it cannot recover from. This may be a fatal connection error received from the Ably service, such as an attempt to connect with an incorrect API key, or some local terminal error, such as that the token in use has expired and the SDK does not have any way to renew it. |
+
+Use the "@current@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/ConnectionStatus.html#current property to check which status a connection is currently in:
+
+```[javascript]
+const connectionStatus = chatClient.connection.status.current;
+```
+
+Use the "@connection.status.onChange()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/ConnectionStatus.html#onChange method to register a listener for status change updates:
+
+```[javascript]
+const { off } = chatClient.connection.status.onStatusChange((change) => console.log(change));
+```
+
+To remove the connection status listener, call the @off()@ function returned in the @subscribe()@ response:
+
+```[javascript]
+off();
+```
+
+Use the "@connection.status.offAll()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/ConnectionStatus.html#offAll method to remove all connection status listeners:
+
+```[javascript]
+chatClient.connection.status.offAll();
+```
+
+h2(#discontinuity). Handle connection discontinuity
+
+If a client briefly loses connection to Ably, for example when driving through a tunnel, the SDK will attempt to recover the connection. If the disruption lasts for less than 2 minutes, then on reconnection the SDK will automatically reattach to any rooms and replay any missed messages.
+
+During periods of discontinuity greater than 2 minutes then you will need to take steps to recover any missed messages, such as by calling "history":/chat/rooms/history.
+
+Each feature of the Chat SDK provides an @onDiscontinuity()@ handler to assist with this. These methods enable you to register a listener that will be notified when discontinuity greater than 2 mins occurs in that features so that you can decide how to handle it.
```

Comment:
```suggestion
Each feature of the Chat SDK provides an @onDiscontinuity()@ handler to assist with this. These methods enable you to register a listener that will be notified when discontinuity occurs in that feature so that you can handle it as needed.
```